instruqt/proxy/01-proxy-config/setup-isolated-host

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -euxo pipefail
+
+echo "Running challenge 01 setup script - Started.."
+epoc_start=`date +%s`
+
+# drop egress (doesn't work with this in track_scripts setup stage as breaks instruqt setup)
+iptables -A OUTPUT -p tcp -m tcp -j DROP
+iptables -A OUTPUT -p icmp -j DROP
+
+# clear some daily package management tasks that cause dpkg lock when installing kurl
+systemctl stop apt-daily.timer
+systemctl disable apt-daily.timer
+systemctl mask apt-daily.service
+systemctl daemon-reload
+mv /usr/lib/apt/apt.systemd.daily /usr/lib/apt/apt.systemd.daily.DISABLED
+rm -f /etc/cron.daily/dpkg 
+rm -f /etc/apt/apt.conf.d/10periodic
```

Comment:
Is this a bug with kURL? I feel like the kurl install script should be able to handle this.

Nice catch by the way!

Reply:
I guess the kurl installer should not crash out when the package manager is busy.
Here I think it's a product of running the install after a fresh provisioning of the instance when these daily apt tasks kick off.  I didn't get this on centos, but needed the larger instance size for the kurl cluster deployment